data/src/test/java/org/apache/iceberg/data/orc/TestGenericData.java

```diff
@@ -237,4 +243,10 @@ private void writeAndValidateRecords(Schema schema, List<Record> expected) throw
       DataTestHelpers.assertEquals(schema.asStruct(), expected.get(i), rows.get(i));
     }
   }
+
+  @Override
+  @Ignore("ORC file format supports null values even for required fields")
```

Comment:
then we shouldn't ignore this test. maybe modify the test behavior instead?

Reply:
Fair point. Changed.